lib/math.ex

```diff
@@ -563,4 +563,123 @@ defmodule Math do
       _ -> raise ArithmeticError, "Inputs are not coprime!"
     end
   end
+
+  # Interpolation
+
+  @doc """
+  Computes the y value of a given t point on a bezier_curve
+
```

Comment:
Explain what happens when points are outside of the range `0..1` (i.e. the difference between this and the bang-version).